geode-cq/src/distributedTest/java/org/apache/geode/security/MultiUserAPIDUnitTest.java

```diff
@@ -226,12 +231,38 @@ public void jsonFormatterOnTheClientWithSingleUser() throws Exception {
     region.put("key", value);
 
     // make sure the client only needs to authenticate once
-    assertThat(TrackableUserPasswordAuthInit.timeInitialized.get()).isEqualTo(1);
+    assertThat(CountableUserPasswordAuthInit.count.get()).isEqualTo(1);
   }
 
+  @Rule
+  public ExecutorServiceRule executor = new ExecutorServiceRule();
```

Comment:
Can this rule be put at the top of the class with the other rules please?

Reply:
done